tests/test_item.py

```diff
@@ -134,6 +134,69 @@ class TestItem(BaseItem):
         self.assertEqual(list(i.keys()), ['keys'])
         self.assertEqual(list(i.values()), [3])
 
+    def test_metaclass_multiple_inheritance_simple(self):
+        class A(Item):
+            save = Field(default='A')
+
+        class B(A): pass
+
+        class C(Item):
+            save = Field(default='C')
+
+        class D(B, C): pass
+
+        self.assertEqual(D(save='X')['save'], 'X')
+        self.assertEqual(D.fields, {'save': {'default': 'A'}})
+
+        # D class inverted
+        class E(C, B): pass
+
+        self.assertEqual(E(save='X')['save'], 'X')
+        self.assertEqual(E.fields, {'save': {'default': 'C'}})
+
+    def test_metaclass_multiple_inheritance_diamond(self):
+        class A(Item):
+            save = Field(default='A')
+
+        class B(A): pass
+
+        class C(A):
+            save = Field(default='C')
+
+        class D(B, C): pass
+
+        self.assertEqual(D(save='X')['save'], 'X')
+        self.assertEqual(D.fields, {'save': {'default': 'C'}})
```

Comment:
I think it lacks another testcase where class D overrides a field defined in class A or B.